test/README.md

```diff
@@ -134,16 +134,16 @@ deactivate
 
 To make the basic test suite run quickly, we use "faked" binary files to test the **CVE mappings**.  However, we want to be able to test real files to test that the **signatures** work on real-world data.
 
-In #99, I've added a _file_test function (to match the existing _binary_test) that takes a url, and package name and a version, and downloads the file, runs the scanner against it, and makes sure it is the file that you've specified.  But we need more tests!
+In #99, I've added a _file_test function (to match the existing _binary_test) that takes a url, and product name and a version, and downloads the file, runs the scanner against it, and makes sure it is the file that you've specified.  But we need more tests!
 
 * Existing tests are in [`test/`](https://github.com/intel/cve-bin-tool/tree/master/test)
 * You can see the scanner tests in ['tests/test_scanner.py'](https://github.com/intel/cve-bin-tool/blob/master/test/test_scanner.py)
-* To add a new test, find an appropriate publicly available file (linux distribution packages and public releases of the packages itself are ideal). You should add the details of the new test case in the `@pytest.mark.parametrize` decorator of test_files test
+* To add a new test, find an appropriate publicly available file (linux distribution products and public releases of the products itself are ideal). You should add the details of the new test case in the `@pytest.mark.parametrize` decorator of test_files test
 * Make sure to hide it behind the LONG_TESTS flag so we aren't doing huge number of downloads for every test suite run
 
 ```python
     @pytest.mark.parametrize(
-        "url, filename, package, version",
+        "url, filename, product, version",
```

Comment:
It is module again and I guess we should keep it
`url, package_name, module, version`.

Reply:
It also sounds good to me.